src/sidebar/components/hypothesis-app.js

```diff
@@ -108,20 +107,6 @@ function HypothesisApp({
     }
   }, [isSidebar, profile, openSidebarPanel, settings]);
 
-  // Show a deprecation warning if current browser is IE11
```

Comment:
Do we show any deprecation warnings now, then?

Reply:
Since last week the sidebar doesn't get created in IE 11. At present there is just a console warning from the boot script. We've discussed in future showing some kind of banner in the LMS app or Via where we know that the user has come to the page in order to annotate. I think Katelyn may have created an issue, I'll have to check.